### ISSUE TITLE
Unable to manually update Atom with `Automatically Update` disabled

### ISSUE THREAD

**Issue by glen-84** [atom/atom#18559]
*Edit by @rsese: I went back a few versions and it looks like the problem starts with ~1.31.0~ 1.32.0 and last worked with ~1.30.0~ 1.31.0*

### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: https://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

I used to be able to update Atom by clicking "Check now" on the About page, when "Automatically download updates" was *unchecked*.

**Before:**

![image](https://user-images.githubusercontent.com/261509/49683355-ef9aba00-facb-11e8-893b-2af3cba4cb87.png)

**Now:**

![image](https://user-images.githubusercontent.com/261509/49683362-05a87a80-facc-11e8-82e2-893ffbe766c1.png)

Why was this changed, or does it have something to do with the x64 version?

### Steps to Reproduce

1. Disable automatic updates

**Expected behavior:** [What you expect to happen]

Still be able to perform a manual update.

**Actual behavior:** [What actually happens]

See the message: "Your system does not support automatic updates".

**Reproduces how often:** [What percentage of the time does it reproduce?]

100%

### Versions

```
Atom    : 1.33.0
Electron: 2.0.11
Chrome  : 61.0.3163.100
Node    : 8.9.3
```

```
apm  2.1.2
npm  6.2.0
node 8.9.3 x64
atom 1.33.0
python
git 2.19.2.windows.1
visual studio 2015
```

OS Name	Microsoft Windows 10 Home
Version	10.0.17134 Build 17134

### Additional Information

n/a

**Comment by sleeve** [comment]
I have also been seeing this on Windows 10 Pro x64 for the past few Atom releases. Thanks for the report and the triage!

**Comment by raduserbanescu** [comment]
If it's of any help, I can confirm that this also happens on **Windows 7 64bit** with **Atom 1.34.0 32bit**

```
Atom    : 1.34.0
Electron: 2.0.16
Chrome  : 61.0.3163.100
Node    : 8.9.3
```

**Comment by glen-84** [comment]
@smashwilson This is still not working in `1.35.1`.

**Comment by smashwilson** [comment]
@glen-84 The fix hasn't made it to stable just yet - the merge commit (b08857c8) is included in 1.36.0-beta0.
